client/src/views/transactions.js

```diff
@@ -0,0 +1,41 @@
+
+import Snabbdom from 'snabbdom-pragma'
+import layout from './layout'
+import { formatSat, formatNumber } from './util'
+
+const staticRoot = process.env.STATIC_ROOT || ''
+const isTouch = process.browser && ('ontouchstart' in window)
+
+const homeLayout = (body, { t, activeTab, ...S }) => layout(
+    <div>
+       { body }
+     </div>
+   , { t, isTouch, activeTab, ...S })
+
+export const recentTxs = ({ mempoolRecent, t, ...S }) => homeLayout(
+    <div className="container">
+      { !mempoolRecent ? <img src="img/Loading.gif" className="loading-delay" />
```

Comment:
Maybe worth extracting the transactions table code into a separate view file that can be shared between here and `dashboard.js`, instead of repeating the code? This will make it easier to make changes later on.

Same for the blocks list.